app/forms/hub/take_action_form.rb

```diff
@@ -88,8 +88,8 @@ def set_default_locale
       @locale = client.intake.locale
     end
 
-    def status_has_changed
-      errors.add(:status, I18n.t("forms.errors.status_must_change")) if status == tax_return&.current_state
+    def state_has_changed
+      errors.add(:state, I18n.t("forms.errors.status_must_change")) if state == tax_return&.current_state
```

Comment:
is there a particular reason to be getting `tax_return&.current_state` instead of `tax_return&.state`? do we always get state from one or the other (the transition or the tax return model)?

Reply:
having now read the rest of the PR, it looks like it's probably best to be reading `tax_return.current_state` since it is a method that checks both! can't remember whether there are places that read directly from the column, just came here to say: using `current_state` is good